python/README.md

```diff
@@ -0,0 +1,12 @@
+# Pure Python example of some rules as code
+
+This implements only the Beard Tax Rules.
+
+This should work in python 3
```

Comment:
```suggestion
This should work in python 3. 

You need to run the tests from within the `python` folder for them to run correctly.
```

Reply:
i wonder why the linter didn't find that